UNITTESTS/target_h/rtos/Semaphore.h

```diff
@@ -1,18 +1,42 @@
 /*
- * Copyright (c) , Arm Limited and affiliates.
- * SPDX-License-Identifier: Apache-2.0
+ * Copyright (c) 2019 ARM Limited
  *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
why are we changing the license for this file?

Reply:
Copy&pasted `Semaphore.h` and removed all but stubs.

Not sure why the Semaphore uses a BSD license instead of Apache.. I'll replace the license back to Apache.

Reply:
fixed.